pmm-app/src/pmm-qan/panel/components/Details/ExplainPlaceholders/QueryFingerprint/QueryFingerprint.utils.ts

```diff
@@ -0,0 +1,17 @@
+import sqlFormatter from 'sql-formatter';
+
+export const replacePlaceholders = (fingerprint: string, placeholders: string[] = []): string => {
+  let replaced = fingerprint || '';
+
+  placeholders.forEach((value, idx) => {
+    const fingerprintIdx = idx + 1;
```

Comment:
the same question, don't understend now what's the magic number and why we need to add this.